src/Acuminator/Acuminator.Utilities/DiagnosticSuppression/SuppressionManager.cs

```diff
@@ -146,13 +170,16 @@ public static IEnumerable<SuppressionDiffResult> ValidateSuppressionBaseDiff()
 
 			var diffList = new List<SuppressionDiffResult>();
 
-			foreach (var entry in Instance._fileByAssembly)
+			lock (Instance._fileSystemService)
```

Comment:
Why do we need this lock?

Reply:
@metallmonster because the VS throws exception about access to file from another thread